backend/test_observer/controllers/reports/reports.py

```diff
@@ -35,6 +35,8 @@
     TestExecution.status,
     TestExecution.review_decision,
     TestExecution.review_comment,
+    TestExecution.c3_link,
```

Comment:
I have also extended the reporting with a c3 link, as it was requested field from the checkbox team in the last SRU review.